vpr/src/route/rr_graph.cpp

```diff
@@ -2392,7 +2392,9 @@ static void check_all_tracks_reach_pins(t_logical_block_type_ptr type,
     VTR_ASSERT(max_chan_width > 0);
 
     int* num_conns_to_track; /* [0..max_chan_width-1] */
```

Comment:
Could recode this as a vector, and just use the 
num_conns_to_track(max_chan_width, 0);  constructor.